app/views/layouts/application.html.erb

```diff
@@ -15,8 +15,14 @@
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
       <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Register as New User", "/register" %>
-      <%= link_to "Login", "/login" %>
+      <% if current_user %>
+        <%= link_to "My Profile", "/profile" %>
```

Comment:
Adding conditionals to show certain things within the nav bar depending on if the user is logged in or not